.github/mergify.yml

```diff
@@ -0,0 +1,8 @@
+pull_request_rules:
+  - name: automatic merge for dependabot pull requests
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
```

Comment:
I assume this will require manual approval before auto-merging.

We need to be more careful with this repo than with compiled languages, as we do not have full code coverage, and UI features sometimes have subtle functional differences.

Reply:
Yes. I removed this requirement, but just reinstated based on your comment.

![image](https://user-images.githubusercontent.com/13387/108416165-9fcce700-722e-11eb-9724-21241a96e3d3.png)